vou/batch.py

```diff
@@ -18,10 +19,9 @@ def load_json(json_file: Path):
 
 
 class BatchSimulation:
-    def __init__(self, params: Path, seeds: Path):
+    def __init__(self, params: Path, distribution_params: Path):
```

Comment:
I think the `params` and `distribution_params` nomenclature could be confusing down the road. Maybe `static_params` and `dynamic_params`? 

That also makes me think about how we can make the `BatchSimulation` class more flexible for future experiments. I wonder if there's a better way to handle `Person` and `Simulation` parameter assignment so we're not hard-coding which parameters are static vs. dynamic.